slug.go

```diff
@@ -193,11 +193,25 @@ func packWalkFn(root, src, dst string, tarW *tar.Writer, meta *Meta, dereference
 			header.Size = info.Size()
 
 		case fm&os.ModeSymlink != 0:
-			target, err := filepath.EvalSymlinks(path)
+			// First read the symlink file to find the destination.
+			target, err := os.Readlink(path)
 			if err != nil {
 				return fmt.Errorf("failed to get symbolic link destination for %q: %w", path, err)
 			}
 
+			// Disallow creating symlinks with absolute paths.
```

Comment:
Since this is a public repo, and this is maybe a breaking change, should we bump the version?

Reply:
I do think we should bump the version. We are still in 0.x on this repo, which gives us some space to make important fixes like this, so I don't think we should for example bump to 1.0 from 0.x to express the breaking change. Going to a new minor within 0.x is maybe reasonable though.

One further thought is that I was looking at this as more of a bug fix, given that if you `Pack()` an archive with an absolute symlink, `Unpack()` will already refuse to unpack it and throw an error. This just moves that validation even earlier. Of course consumers of this package could be using it for only the `Pack()` function and not really looking at it the same way, so in principle I agree with you.

Reply:
Yeah, that's why I said _maybe_ a breaking change. Since it sounds like the only thing it's "breaking" is something that was already broken anyway, agreed that a major bump definitely does not make sense. Even a patch version may be reasonable, but I'd probably lean toward minor